front/app/hooks/useIdeasFilterCounts.ts

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+
+// services
+import {
+  IIdeasFilterCounts,
+  ideasFilterCountsStream,
+  IIdeasFilterCountsQueryParameters,
+} from 'services/ideas';
+
+// utils
+import { omitBy, isNil } from 'lodash-es';
+
+// typings
+import { NilOrError } from 'utils/helperUtils';
+
+export default function useIdeasFilterCounts(
+  props: IIdeasFilterCountsQueryParameters | null
```

Comment:
`null` is not needed here.

We can do this in IdeasWithFiltersSidebar:
`    <GetIdeasFilterCounts queryParameters={ideas.queryParameters}>
` 
instead of 
`    <GetIdeasFilterCounts queryParameters={get(ideas, 'queryParameters', null)}>
`.

Reply:
Much nicer indeed!